### PR TITLE
e2e(react-hmr sample)

### PR DESCRIPTION
**Pull request by PeaceDate**
This PR is created instead of Vasiliy PR ---> https://github.com/module-federation/module-federation-examples/pull/2550

Screenshot 2023-01-03 at 18 54 55

### CODE REVIEW

File: cypress/common/base.ts [hunk 1]
```diff
@@ -620,6 +632,32 @@ export class BaseMethods {
...
+        return cy.get(selector)
+            .each((element: JQuery<HTMLElement>) => {
+                if(element.text() === text) {
+                    expect(element.attr('href')).to.be.eq(link)
+                    expect(element.is(':disabled')).to.be.eq(false)
+                }
+            });
+    }
+
+    public checkParentElementWithTextContainsLink(
```
Comment (edchai) [inline cypress/common/base.ts]: merge it with previous one

File: react-hmr/e2e/checkHostApp.cy.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import { BaseMethods } from '../../cypress/common/base';
+import { baseSelectors } from '../../cypress/common/selectors';
+import { Constants } from '../../cypress/fixtures/constants';
+import { CssAttr } from '../../cypress/types/cssAttr';
+
+
```
Comment (edchai) [inline react-hmr/e2e/checkHostApp.cy.ts]: remove blankline

File: react-hmr/e2e/checkHostApp.cy.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        pageContent: '',
+        link: Constants.hrefs.reactHmrHomeLink
+    },
+    {
+        linkName: 'Button',
+        pageContent: Constants.elementsText.reactHmrHostButtonPageText,
+        link: Constants.hrefs.reactHmrButtonLink
+    },
+    {
+        linkName: 'Heading',
```
Comment (edchai) [inline react-hmr/e2e/checkHostApp.cy.ts]: link names to constants

**Review by edchai — CHANGES_REQUESTED** [review]

File: cypress/common/base.ts (hunk 1 above)
Comment (PeaceDate) [reply cypress/common/base.ts]: Fixed

File: react-hmr/e2e/checkHostApp.cy.ts (hunk 2 above)
Comment (PeaceDate) [reply react-hmr/e2e/checkHostApp.cy.ts]: Fixed

File: react-hmr/e2e/checkHostApp.cy.ts (hunk 3 above)
Comment (PeaceDate) [reply react-hmr/e2e/checkHostApp.cy.ts]: Fixed

**Review by edchai — APPROVED** [review]

**Pull request merged**